docs/reference/structure-extensions.adoc

```diff
@@ -0,0 +1,137 @@
+:m3: https://lionweb-io.github.io/specification/metametamodel/metametamodel.html
+:mpshelp: https://www.jetbrains.com/help/mps
+:underbracket: &#9141;
+
+= Reference for LionWeb Extensions on MPS' Language Structure Aspect
+:experimental:
+
+LionWeb can represent MPS languages in LionCore.
+It supports most MPS languages without changes to the language.
```

Comment:
“most”: what are the limitations?

Reply:
What's listed here (plus one I'll add tomorrow). I'll adjust the text.